torch/csrc/autograd/variable.h

```diff
@@ -377,9 +377,12 @@ struct TORCH_API AutogradMeta : public c10::AutogradMetaInterface {
 /// - NO_GRAD_MODE should be set when a view in created when GradMode is disabled
 /// - MULTI_OUTPUT_NODE should be set when a Node created by codegen code returns
 ///   multiple differentiable views
+/// - MULTI_OUTPUT_SAFE should be set when a view was returned by a function
```

Comment:
Could you add here: These are note considered as views for now to for the view+inplace logic! The graph won't be rewritten when an inplace is done, only a warning will be thrown.